storm-core/src/clj/backtype/storm/zookeeper.clj

```diff
@@ -103,11 +103,14 @@
      (catch Exception e (throw (wrap-in-runtime e))))))
 
 (defnk delete-node
```

Comment:
Oh, I got it. Whole usages are calling zk/delete-recursive so your changeset makes sense. My mistake.